config/routes.rb

```diff
@@ -44,6 +44,7 @@
       resources :vote_users, path: :motesanvandare, controller: :vote_users do
         get 'change_state', on: :member
         get 'make_all_not_present', on: :collection
+        get 'new_votecode', on: :member
```

Comment:
I feel like these actions should be `patch`.

- `GET` is mostly used for loading a page
- `POST` for creating new content
- `PATCH` for updating existing content